accelerator/http.c

```diff
@@ -245,11 +223,19 @@ static int ta_http_header_iter(void *cls, enum MHD_ValueKind kind,
   return MHD_YES;
 }
 
-static int ta_http_handler(void *cls, struct MHD_Connection *connection,
-                           const char *url, const char *method,
-                           const char *version, const char *upload_data,
-                           size_t *upload_data_size, void **ptr) {
-  int ret = MHD_NO;
+static int request_log(void *cls, const struct sockaddr *addr, socklen_t addrlen) {
+  char buf[30];
+  struct sockaddr_in *addr_ip = (struct sockaddr_in *)addr;
+  char *ip = inet_ntoa(addr_ip->sin_addr);
+  time_t rawtime = time(NULL);
+  strftime(buf, 30, "%c", localtime(&rawtime));
+  printf("%s  - -  [%s]", ip, buf);
```

Comment:
Why did you dump IP address as the necessary column?

Reply:
I took the output of served as example, see #226 .
The output included the IP address of incoming requests.